src/components/side-panel/side-panel.tsx

```diff
@@ -28,26 +34,27 @@ export const SidePanel = ({ offset, onSubmit }: IProps) => {
         toggleSubmitting,
     } = useBuilder();
     const { showToast } = useDisplay();
+    const element = focusedElement?.element;
+    const schema = SchemaHelper.buildSchema(EElementType.TEXTAREA);
```

Comment:
these are probably test code, let's follow the latest implementation from main branch